src/Pim/Component/Catalog/Normalizer/Indexing/Product/TextNormalizer.php

```diff
@@ -30,6 +30,12 @@ public function supportsNormalization($data, $format = null)
      */
     protected function getNormalizedData(ProductValueInterface $productValue)
     {
-        return $productValue->getData();
+        $textValue = $productValue->getData();
```

Comment:
In this case, is it usefull to check `getData` is null, as we will just return `getData` as it is, or `null` (so also `getData` as it is)?